tests/repository_data/client/test_repository1/metadata/current/targets.json

```diff
@@ -40,7 +40,7 @@
   "targets": {
    "file1.txt": {
     "custom": {
-     "file_permissions": "0644"
+     "file_permissions": "0664"
```

Comment:
Why did the write bit get added to these?

Reply:
Good catch, Marina! Also, do we discuss anywhere why we set file permissions here at all? Is it show what the "custom" attribute can do? If so, we really ought to document it somewhere.

Reply:
> Why did the write bit get added to these?

Nice catch!

664 is the default file permissions for user-created files on Fedora (my host OS) and permissions are not explicitly set on the target files at creation time. Presumably this differs from wherever the original files were generated.

I can edit the patch to drop this change?

Reply:
Default file permissions on Ubuntu and Debian are 644 (umask `0022` vs umask `0002` on Fedora), which I believe would account for the discrepancy. I'll edit the patch to drop this change.

Reply:
I added an extra patch efae583 to set the permissions on `file1.txt` so that we don't generate different data on different Linux host OS.